homeassistant/components/totalconnect/__init__.py

```diff
@@ -24,30 +26,56 @@
     extra=vol.ALLOW_EXTRA,
 )
 
-TOTALCONNECT_PLATFORMS = ["alarm_control_panel", "binary_sensor"]
+PLATFORMS = ["alarm_control_panel", "binary_sensor"]
+
 
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Old way to set up."""
+    return True
```

Comment:
I've added the breaking change label.

In the description you've added that configuration.yaml is no longer required, but this code does not support importing it.

Reply:
I'll go with breaking change.  Updated the PR info above and the linked documentation PR.